package.json

```diff
@@ -89,6 +89,7 @@
     "@metaplex-foundation/js": "^0.20.1",
     "@react-spring/web": "^9.7.4",
     "@reduxjs/toolkit": "^1.9.7",
+    "@reservoir0x/relay-sdk": "^1.6.12",
```

Comment:
Why do we need this? We're not using the SDK:

<img width="1202" alt="image" src="https://github.com/user-attachments/assets/504040ea-040d-435f-a465-f0415322ec7d" />

The only thing we consume is `Execute` type, which could be copied over from Execute.d.ts.